### DISCUSSION TITLE
Why is it recommended to install vue-leaflet as a dev dependency?

### DISCUSSION THREAD

**Discussion by ruilvo** [vue-leaflet/vue-leaflet#351] [General]
I noticed the readme.me suggests to install the library with:

```sh
npm i -D @vue-leaflet/vue-leaflet leaflet
```

Why is vue-leaftlet installed as a dev dependency?

**Comment by biblia19** [comment]
I think It is beta version.
https://github.com/vue-leaflet/vue-leaflet/blob/db34dff79cc62bc6fa51357e953e9bcf55725c94/README.md?plain=1#L5-L7
